Moodlet/app/src/main/java/com/cmput3owo1/moodlet/models/User.java

```diff
@@ -6,8 +6,10 @@
  */
 public class User {
     private String username;
-    private String fullName;
+    private String fullname;
```

Comment:
I think camel casing was correct before since "full name" would be two words whereas username is just one. Leaving it as is would also be fine.

Reply:
I did this cuz firebase didn't recognize "fullName" because in the datebase its "fullname" The better solution is probably to change it in firestore.

Reply:
I'd prefer the fix to be in Firestore as well but for the sake of getting this PR in, I'll open a new issue to track it.

Reply:
I can just do it as part of this PR, it shouldn't take long